### DISCUSSION TITLE
Welcome to Attendance-management-system-Devops-lab- Discussions!

### DISCUSSION THREAD

**Discussion by KhushiJain2004** [KhushiJain2004/Attendance-management-system-Devops-lab-#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by KhushiJain2004** [comment]
Hello everyone
We are part of the same team for our Devops project 3
Our topic is attendance management system
And I will be leading the team as decided by all of us in the lab.

**Comment by Kashishturan** [comment]
Hey, everyone
It's nice to be working with you all,  I am posting the few basic points over here which we need to keep in mind during our project:
Project 3
- 5 to 8 team members
- Two lab turns ( 1st turn - demo (7nov) , 2nd turn - D-Day (21nov) )
- Dates will be told later
- No restriction wrt tools or lang ( wildcard - unknown risk will be given )
- Need to have a team lead
Demo
The team decides which running functional part u would like to show 
Prototype

- Like if ur project have 3 modules
- Show 1 module with its element
- Can be 1/4 or 1/8 of ur project
- Show the best part or best highlight of our project

**Comment by akshit-bhutani-19** [comment]
Hello

**Comment by KhushiJain2004** [comment]
guys this is the home page design i've made .. if you have any suggestions do let me know 
<img width="906" alt="image" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/c5f69825-1dc5-4e27-bdcf-ec19873e4d6b">

**Comment by KhushiJain2004** [comment]
<img width="908" alt="image" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/42b84949-3a22-443c-80da-4731112e3f9b">

**Comment by KhushiJain2004** [comment]
<img width="911" alt="image" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/9dfba483-f5b0-4caa-86de-ffd74df2e77c">

<img width="911" alt="image" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/200f12a0-da42-4714-b8e6-153f214cf1d1">

**Comment by KhushiJain2004** [comment]
These cards acts as buttons that will redirect the user to the respective login page when clicked ..

**Comment by Kashishturan** [comment]
@KhushiJain2004 it look nice but rather than click option we should add text as it opens to make it more clear .

**Comment by Kashishturan** [comment]
others also tell what you guys think.

**Comment by Aaryanb45** [comment]
So i have made the web page after Khushi output

**Comment by Aaryanb45** [comment]
<img width="1280" alt="Screenshot 2023-11-02 at 10 00 42 PM" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/113382989/e763e568-cc66-4c68-b2f9-92f081d0d321">
Here's the output

**Reply by Kashishturan** [reply]
yes it look good , it have the elements we need just some adding will be done like those in khushi's output.

**Comment by Aaryanb45** [comment]
I would be making some changes in this web page by putting some css into it.
I just wanted everyone review is this sample is correct or not

**Comment by akshit-bhutani-19** [comment]
We don't need Role in Login page coz that we would get from Khushi's Page. Rest both look good

**Comment by Kashishturan** [comment]
So guys in today's meet everyone got there part to do 
I am posting it here please approve the message

**Comment by Kashishturan** [comment]
So @Aryan have to make the login pages.
@KhushiJain2004 have to make the dashboard
@Kashishturan got to make add new things option pages
@Dhruv-Jain31 You have to make edit option pages
@kanavpal You have to make new classes, faculty and student pages

**Reply by Aaryanb45** [reply]
ok

**Comment by Kashishturan** [comment]
@rajatkamboj06 you were not present in meeting
You have to make nav bar for the login pages

**Comment by Kashishturan** [comment]
.@Akki019 and @Palakgupta26 you guys are working on backend part.

**Comment by Kashishturan** [comment]
![WhatsApp Image 2023-11-04 at 23 44 24_f96b5686](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/142867357/e5dcb916-76f3-4862-8e93-ead5daccf369)

**Comment by Kashishturan** [comment]
![WhatsApp Image 2023-11-04 at 23 44 24_9213883f](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/142867357/27d3410d-7bb2-40c8-9f6b-0b00023a249f)

**Comment by KhushiJain2004** [comment]
Guys we will be displaying this admin module of our project as demo on the next lab

**Comment by KhushiJain2004** [comment]
If there is any confusion or problem regarding the work .. discuss it here ..

**Comment by Dhruv-Jain31** [comment]
mine 2 templates are ready pls review guys.
<img width="724" alt="Screenshot 2023-11-05 022634" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/08a39b6f-9a5f-41e5-ace4-4c5b6485f0f9">
<img width="960" alt="Screenshot 2023-11-05 022700" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/fe9bb757-f628-49bc-b2f8-c2f9849e3ceb">

**Comment by KhushiJain2004** [comment]
![WhatsApp Image 2023-11-05 at 09 17 14_3bb4a548](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/fe4071f2-d8e7-4825-b123-621551208530)

This is the admin dashboard i've created ..please review

**Comment by KhushiJain2004** [comment]
Guys what is the progress ..  how much work is completed?
Have you all completed the pages assigned to you ?

**Reply by Dhruv-Jain31** [reply]
i have completed my 2 templates working on the 3rd one.

**Comment by Dhruv-Jain31** [comment]
hey guys last template is completed as well. pls review
<img width="951" alt="Screenshot 2023-11-05 194927" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/5a023bec-29fb-40ef-b85c-740e6318b0d3">

**Comment by Dhruv-Jain31** [comment]
updated the attendance templates as per the requirements
<img width="960" alt="Screenshot 2023-11-05 203358" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/a6603506-7b68-4315-bdf9-32b7ccb0af8f">

**Comment by Dhruv-Jain31** [comment]
done with the last template as well. pls review
<img width="960" alt="Screenshot 2023-11-05 205552" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/dee4e987-60d4-4b44-a0ae-f24e54523864">

**Reply by KhushiJain2004** [reply]
yupp this is good ..

**Comment by Aaryanb45** [comment]
<img width="1280" alt="Screenshot 2023-11-06 at 3 15 57 AM" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/113382989/5bd3ff42-9b7e-4321-b8f8-92a150f4bb40">
I hv made changes please review it and tell me in the morning if any changes to be made

**Comment by KhushiJain2004** [comment]
Division of work for the teacher and student dashboard will be as follows : 

![WhatsApp Image 2023-11-13 at 15 45 57_b8ba766e](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/1eb45388-8ca6-4faf-9ead-d5e137776ac7)

 Dhruv you have to make these two pages ..

**Comment by KhushiJain2004** [comment]
![WhatsApp Image 2023-11-13 at 15 45 21_2fe091e3](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/50b7cd61-0b45-4817-8d3d-d4fa5a7dfa72)
 
Kashish these 2 pages are yours

**Reply by Kashishturan** [reply]
okay

**Comment by KhushiJain2004** [comment]
![WhatsApp Image 2023-11-13 at 15 44 48_61726697](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/3a7a8cbb-f472-4796-ba50-c37a8eeb8126)

![WhatsApp Image 2023-11-13 at 15 44 57_945770cb](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/1302f65f-b598-4feb-86ba-adb5958dbc45)

Kanav you have to make these pages

**Comment by KhushiJain2004** [comment]
![image](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/ab4e70f0-8559-44b4-bcda-6868efd6b4c2)

![image](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/c8a1bb72-5578-4fe4-8bce-bb9533836d78)

Aryan these are your pages

**Comment by KhushiJain2004** [comment]
i will be making the main dashboard pages and an attendance view page

**Comment by KhushiJain2004** [comment]
please update when you are done with your respective pages

**Comment by Kashishturan** [comment]
Yup will let you know @KhushiJain2004

**Comment by KhushiJain2004** [comment]
Rajat you have to make this page.. it will be a part of the teacher dashboard that will let them mark a student present or absent on a particular date .. build this page accordingly .. if there is any confusion you can discuss it here

![WhatsApp Image 2023-11-14 at 19 25 02_4650e74c](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/129265117/a7e07803-d6e9-40ae-8f61-e6d5881cc053)

**Comment by rajatkamboj06** [comment]
<img width="1433" alt="Screenshot 2023-11-14 at 8 13 38 PM" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/142865956/7e08a3d3-8dd4-479c-aadb-c4d78285eed6">
It is okay or should make any change.

**Reply by KhushiJain2004** [reply]
some changes are required rajat .. this page needs to have a table representing the attendance of the student (datewise)... as shown in the layout i posted

**Reply by KhushiJain2004** [reply]
also a toggle button or check boxes for the teacher to edit attendance

**Comment by Kashishturan** [comment]
![IMG-20231115-WA0095.jpg](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/142867357/864bb7fb-7098-4a49-87a3-d1e446ab14ac)

**Comment by Kashishturan** [comment]
![IMG-20231115-WA0096.jpg](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/142867357/b44b5208-8e7a-4376-8f7a-af31399464a2)

**Comment by Kashishturan** [comment]
Guys above are the ss of view pages output... Suggest changes if required..

**Comment by Aaryanb45** [comment]
I have made the page of teacher dashboard

**Comment by Aaryanb45** [comment]
![PHOTO-2023-11-15-22-09-12](https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/113382989/398b7e75-1b3d-4270-a2cd-bb8972bbf2d5)

**Comment by Aaryanb45** [comment]
<img width="1280" alt="Screenshot 2023-11-16 at 6 10 18 PM" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/113382989/21215e44-ad07-432b-87d9-d838dcdbcc50">

**Comment by Aaryanb45** [comment]
guys review on this and let me know if any changes to be dn..

**Comment by rajatkamboj06** [comment]
<img width="1433" alt="Screenshot 2023-11-16 at 6 21 16 PM" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/142865956/ba3feeda-fdff-41b6-a971-3921e499a800">
Teacher dashboard that will let them mark a student present or absent on a particular date.

**Comment by Dhruv-Jain31** [comment]
<img width="958" alt="Screenshot 2023-11-17 131039" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/d8e3d404-ff4b-44b6-9c6e-a84421e2b398">
Guys pls review.

**Comment by Dhruv-Jain31** [comment]
<img width="954" alt="Screenshot 2023-11-17 133536" src="https://github.com/KhushiJain2004/Attendance-management-system-Devops-lab-/assets/126942955/34093eb6-8a67-4f75-b34b-f55e5b4cd2c8">
Done with the second ss as well. Guys pls review
